var/spack/repos/builtin/packages/llvm-amdgpu/package.py

```diff
@@ -32,6 +33,7 @@ class LlvmAmdgpu(CMakePackage):
     depends_on('python', type='build')
     depends_on('z3', type='link')
     depends_on('zlib', type='link')
+    depends_on('unzip', type='link')
```

Comment:
```suggestion
```
unzip is a binary, not sure what it does here? can you comment on this?

Reply:
i have made the changes per your suggestions.